### ISSUE TITLE
  Hi, our lab created (a long time ago)...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#5071]
*Submitted from the feedback form on the WormBase website.*

<p>&nbsp;&nbsp;Hi, our lab created (a long time ago) a cDNA library and I PCR amplified FARL-11 from it, and saw a 6 bp (2 amino acid) insertion at the N-terminus that does not match the cDNA sequence that is listed on wormbase. <br/>Would it be wise to drop out those 6 base pairs to match wormbase's listed sequence? <br/>Thanks for the help. </p>

**Reported by:** scma**************
**Submitted from:** <a target="_blank" href="http://www.wormbase.org//tools/support">/tools/support</a>
**Browser:** Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36

**Comment by raymond91125** [comment]
Hi,

I'm not sure what you mean by "Would it be wise to drop out those 6 base 
pairs ...". However, if you tell us what the whole cDNA sequence you saw 
is, perhaps we can assess if we should extend our gene model or tell you 
that it is something novel.

Thanks.

Raymond

PS. I don't have user email.

**Comment by gw3** [comment]
Hi,

Yes we can see what you are reporting.
The first intron of farl-11 has alternate acceptor splice sites differing
by 6 bases.
These splice site appear to be used with approximately equal frequencies.

We will create a new isoform, so this locus will have two CDS sequences
(F10E7.8a and F10E7.8b) in the release WS257.

It would help if you could submit any cDNA sequences you have to INSDC (
http://www.insdc.org/). That way the sequence will be automatically added
in to the WormBase database and changes to the gene structure can be easily
attributed to your work.

Gary Williams
WormBase

**Comment by gw3** [comment]
Hi Gary,

This is interesting.

I took a look back at all the FARL-11 clones I pulled from our cDNA library, and have yet to find the "canonical" farl-11 sequence, as all my clones have the 6bp exon.

Any reasons for the discrepancy? Or just random chance I have yet to pull the "canonical" sequence?

Thanks.
Sterling.
________________________________

**Comment by gw3** [comment]
Sterling,

I have got all published RNASeq data for C.elegans from the NCBI Short 
Read Archive.
I then aligned these reads to the genome.
Looking at farl-11, the first intron has approximately equal numbers of 
reads that span the intron using each of the alternate acceptor splice 
sites.

I have been assuming that the splicing we see in the RNASeq data is 
representative of the cDNA sequences (which the RNASeq data is created 
from).
I cannot give a good explanation for the differences between what you 
find in your cDNA and the RNASeq data.

I can image some unlikely explanations (you are growing the worms under 
stressful conditions leading to a biased splicing, you have an unusual 
strain of worms, the RNASeq data comes from many dozens of projects with 
different conditions and many of these conditions have lead to biased 
splicing, there is a region of another gene in C. elegans that has 
nearly the same sequence except for these 6 bases and reads expressed 
from this other gene are being aligned to farl-11 giving a false 
indication of alternate splicing).

The last of these possibilities is a distinct possibility, but I've just 
tried searching against the C. elegans genome with a query sequence made 
from short regions either side of the intron with the 6 bases removed. 
The only region in the genome that the query sequence matches is 
farl-11, so I think that possibility can be excluded.

We will put both isoforms in the database as a possibility, for now.

Don't forget to submit your sequences to INSDC :-)

Thanks,
Gary
